### ISSUE TITLE
dev-cpp/wangle-2016.11.28.00 missing /include/folly/config.h

### ISSUE THREAD

**Issue by archenroot** [facebook/wangle#78]
While installed folly (latest from github as of now) against wangle-2016.11.28.00 I encountered missing  /include/folly/config.h which is installed as  /include/folly/folly-config.h

I just tested if that is correct file by creating symlink and after link in place installation finished successfully.

**Comment by anirudhvr** [comment]
Is this fixed with the latest wangle? If so, please close out.

**Comment by archenroot** [comment]
I am working in the moment non multipath networking for torch, so will update this one later.

**Comment by archenroot** [comment]
I tried today to compile again this version of wangel and finished with another error:
```
[ 28%] Performing build step for 'gtest'
cd /tmp/portage/dev-cpp/wangle-2016.11.28.00/work/wangle-2016.11.28.00_build/gtest/src/gtest-build && /usr/bin/cmake -Dmake=make -P /tmp/portage/dev-cpp/wangle-2016.11.28.00/work/wangle-2016.11.28.00_build/gtest/src/gtest-stamp/gtest-build-Gentoo.cmake
[ 29%] Building CXX object CMakeFiles/wangle.dir/concurrent/ThreadedExecutor.cpp.o
/usr/bin/x86_64-pc-linux-gnu-g++   -I/tmp/portage/dev-cpp/wangle-2016.11.28.00/work/wangle-2016.11.28.00/wangle/.. -I/tmp/portage/dev-cpp/wangle-2016.11.28.00/work/wangle-2016.11.28.00_build/gtest/src/gtest/googlemock/include -I/tmp/portage/dev-cpp/wangle-2016.11.28.00/work/wangle-2016.11.28.00_build/gtest/src/gtest/googletest/include   -DNDEBUG -march=skylake -mabm -maes -madx -mrtm -mavx256-split-unaligned-load -mavx256-split-unaligned-store -O2 -pipe   -std=c++1y -fPIC -o CMakeFiles/wangle.dir/concurrent/ThreadedExecutor.cpp.o -c /tmp/portage/dev-cpp/wangle-2016.11.28.00/work/wangle-2016.11.28.00/wangle/concurrent/ThreadedExecutor.cpp
In file included from /tmp/portage/dev-cpp/wangle-2016.11.28.00/work/wangle-2016.11.28.00/wangle/concurrent/ThreadedExecutor.cpp:11:0:
/tmp/portage/dev-cpp/wangle-2016.11.28.00/work/wangle-2016.11.28.00/wangle/../wangle/concurrent/ThreadedExecutor.h:76:36: error: field ‘stopping_’ has incomplete type ‘std::atomic<bool>’
   std::atomic<bool> stopping_{false};
                                    ^
In file included from /usr/lib/gcc/x86_64-pc-linux-gnu/6.2.0/include/g++-v6/bits/shared_ptr_atomic.h:33:0,
                 from /usr/lib/gcc/x86_64-pc-linux-gnu/6.2.0/include/g++-v6/memory:83,
                 from /tmp/portage/dev-cpp/wangle-2016.11.28.00/work/wangle-2016.11.28.00/wangle/../wangle/concurrent/ThreadedExecutor.h:16,
                 from /tmp/portage/dev-cpp/wangle-2016.11.28.00/work/wangle-2016.11.28.00/wangle/concurrent/ThreadedExecutor.cpp:11:
/usr/lib/gcc/x86_64-pc-linux-gnu/6.2.0/include/g++-v6/bits/atomic_base.h:126:12: note: declaration of ‘struct std::atomic<bool>’
     struct atomic;
            ^~~~~~
CMakeFiles/wangle.dir/build.make:446: recipe for target 'CMakeFiles/wangle.dir/concurrent/ThreadedExecutor.cpp.o' failed
make[2]: *** [CMakeFiles/wangle.dir/concurrent/ThreadedExecutor.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[2]: Leaving directory '/tmp/portage/dev-cpp/wangle-2016.11.28.00/work/wangle-2016.11.28.00_build'
CMakeFiles/Makefile2:484: recipe for target 'CMakeFiles/wangle.dir/all' failed
make[1]: *** [CMakeFiles/wangle.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
-- gtest build command succeeded.  See also /tmp/portage/dev-cpp/wangle-2016.11.28.00/work/wangle-2016.11.28.00_build/gtest/src/gtest-stamp/gtest-build-*.log
cd /tmp/portage/dev-cpp/wangle-2016.11.28.00/work/wangle-2016.11.28.00_build/gtest/src/gtest-build && /usr/bin/cmake -E touch /tmp/portage/dev-cpp/wangle-2016.11.28.00/work/wangle-2016.11.28.00_build/gtest/src/gtest-stamp/gtest-build
[ 30%] No install step for 'gtest'
cd /tmp/portage/dev-cpp/wangle-2016.11.28.00/work/wangle-2016.11.28.00_build/gtest/src/gtest-build && /usr/bin/cmake -E echo_append
cd /tmp/portage/dev-cpp/wangle-2016.11.28.00/work/wangle-2016.11.28.00_build/gtest/src/gtest-build && /usr/bin/cmake -E touch /tmp/portage/dev-cpp/wangle-2016.11.28.00/work/wangle-2016.11.28.00_build/gtest/src/gtest-stamp/gtest-install
[ 32%] Completed 'gtest'
/usr/bin/cmake -E make_directory /tmp/portage/dev-cpp/wangle-2016.11.28.00/work/wangle-2016.11.28.00_build/CMakeFiles
/usr/bin/cmake -E touch /tmp/portage/dev-cpp/wangle-2016.11.28.00/work/wangle-2016.11.28.00_build/CMakeFiles/gtest-complete
/usr/bin/cmake -E touch /tmp/portage/dev-cpp/wangle-2016.11.28.00/work/wangle-2016.11.28.00_build/gtest/src/gtest-stamp/gtest-done
make[2]: Leaving directory '/tmp/portage/dev-cpp/wangle-2016.11.28.00/work/wangle-2016.11.28.00_build'
[ 32%] Built target gtest
make[1]: Leaving directory '/tmp/portage/dev-cpp/wangle-2016.11.28.00/work/wangle-2016.11.28.00_build'
Makefile:138: recipe for target 'all' failed
make: *** [all] Error 2

```

This happened now after I upgraded a dependency FOLLY library from GIT master branch, so I will take a newest release of wangle, make new Gentoo ebuild and give it a try again.

**Comment by archenroot** [comment]
Ok, I confirm that in the moment I am able to compile the latest version:
`=dev-cpp/wangle-2017.01.02.00 ~amd64`
